### PR TITLE
German translation update

### PR DESCRIPTION
**Pull request by Lordsem**

### CODE REVIEW

**Comment by DexterHuang** [comment]
hello seems like there is an error, could you try to run it pass a JSON validator and fix the error :)

**Comment by Lordsem** [comment]
Error fixed 👍🏻

File: contribution/lang/de.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
  	 	},
  	 	"store": {
  	 	 	"name": {
- 	 	 	 	"trans": "Trinoky Markt",
+ 	 	 	 	"trans": "Trinoky Mart",
```
Comment (JARUSZKA) [inline contribution/lang/de.json:89]: Markt = Mart

Comment (JARUSZKA) [reply contribution/lang/de.json:89]: So the correct german line is
Trinkony Markt

**Pull request merged**